Documentation/Contributing/ci-configuration.md

```diff
@@ -0,0 +1,13 @@
+---
+title: CI Configuration
```

Comment:
I think these env vars for the CI are fairly self explanatory. Do we need this help topic?

Reply:
@travisn Yes, this document accidentally slipped in. The reason for this document is that even though the env vars are self explanatory, there is not a (complete) list of all the env vars you would need to run all CI flows in a fork/customized repo.

I think it in general makes sense to have such a page with CI details to make it easier to "self host" the CI. What do you think? Should I remove it from this PR/commit?

Reply:
Would you mind opening a separate PR for this?